sos/report/__init__.py

```diff
@@ -421,9 +421,9 @@ def display_help(cls, section):
             'Plugins may specify different collections on different distribu'
             'tions, and some plugins may only be for specific distributions. '
             'Distributions are represented within SoS by \'policies\' and may '
-            'influence how other SoS commands or options function. For example'
-            'policies can alter where the --upload option defaults to or '
-            'functions.'
+            'influence how other SoS commands or options function. For '
+            'example policies can alter where the --upload option defaults '
+            'to or functions.'
```

Comment:
This doesn't look like a typo to me, it's still the same tring 🤷🏽

Reply:
It should be the space between example and policies. If you run 'sos help report' you get:

```
For examplepolicies can alter where the --upload option defaults
to or functions.
```

That this change should fix.

Reply:
Yup, I see it now